src/bp/ui-shared-lite/auth/index.ts

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+
+import { AxiosInstance } from 'axios'
+import { StoredToken, TokenUser, TokenResponse } from 'common/typings'
+import moment from 'moment'
+import ms from 'ms'
+import storage from '../utils/storage'
+
+export const TOKEN_KEY = 'bp/token'
+
+const MIN_MS_LEFT_BEFORE_REFRESH = ms('5m')
+
+export const getToken = (onlyToken: boolean = true): StoredToken | string | undefined => {
```

Comment:
why is all this in share-lite ? do you need it in the webchat somehow ?

Reply:
Channel-web loads the lite edition of the studio, if I put this code in ui-shared, lite will load the full package (including blueprint), increasing dramatically the bundle size. So the workaround is shared lite, for now

Reply:
This was my question, do you actually need this in webchat ?

Reply:
Yes. The webchat doesn't know how to talk to the backend, it uses a lite bundle of the studio, which sets up the communication channel with the backend, and provides basic features, like setting up the websocket connection